apps/authentication/routes.py

```diff
@@ -139,6 +138,32 @@ def register():
     else:
         return render_template('pages/register.html', form=create_account_form)
 
+@blueprint.route('/groups/create', methods=['GET', 'POST'])
+def create_group():
+    form = GroupForm()
+    students = Users.query.filter_by(category='student').all()
```

Comment:
I'm assuming this variable `students`  will be the candidates for group membership, right? if that is true, not only students can be members but any VALID user except the users on the category "user". Can you please double check this point? and possibly adjust the filter + rename the variable.